.github/workflows/ci_workflows.yml

```
@@ -134,3 +134,39 @@ jobs:
       test_command: pytest -p no:warnings --astropy-header -m "not hypothesis" -k "not test_data_out_of_range and not test_set_locale and not TestQuantityTyping" --pyargs astropy
       targets: |
         - cp311-manylinux_x86_64
+
+
+  test_limited_api_build:
+
+    # Test to make sure that we can build astropy with the limited API
+    # Currently, several extensions do not compile, so we remove them for
+    # now to make sure the remaining ones build fine. Once all the exceptions
+    # are removed, we can remove this job and instead opt in to the limited
+    # API in one of the main build+test tox jobs.
+
+    runs-on: ubuntu-latest
+    name: Python 3.11 (build only) with limited API
```

To avoid disappointments :)

```suggestion
    name: Python 3.11 (build only, partial) with limited API
```